test/utils/FakeObjects.scala

```diff
@@ -230,4 +230,5 @@ trait FakeObjects {
 
   val agentDetails = AgentDetails(name = "Awesome Agent", address = "1 Awesome Street, AB1 1BA")
 
+  val propertyLinksCount = 1
```

Comment:
Not worth having here, probably better to have in the test.

Reply:
Moved to test case